content/security/reporting.adoc

```diff
@@ -29,6 +29,8 @@ Please report issues present in the following components:
 * Jenkins, including installers and Docker images published by the Jenkins project
 * Jenkins plugins published by the Jenkins project (listed on https://plugins.jenkins.io/[plugins.jenkins.io] and/or hosted in https://github.com/jenkinsci[the jenkinsci GitHub organization])
 * Additional components published by the Jenkins project for general use, such as Docker images
+* Jenkins infrastructure projects, such as https://www.jenkins.io/[jenkins.io] or more generally the repositories hosted in the https://github.com/jenkins-infra[jenkins-infra GitHub organization]
+** The infrastructure security reports will be triaged by the Jenkins security team, but worked on by the link:/project/team-leads/#infrastructure[Infrastructure Officer] and their team
```

Comment:
Why would a reporter care who process these reports though? I don't understand the purpose of this change, it just duplicates the security-related content of https://www.jenkins.io/project/team-leads/#infrastructure from when the split of responsibilities was clarified.

It also directly contradicts https://github.com/jenkins-infra/.github/pull/21#issuecomment-1279718131

Reply:
AFAICT the infra team does not receive vulnerability reports.

@dduportal do you have something in place to receive them yourself or do you rely on the SECURITY tracker?

Reply:
The purpose of this change is to be consistent with the https://github.com/jenkins-infra/.github/pull/21 so that reporters that are reading the doc, does not find the fact that we care only about Jenkins core + plugins and not about the infra

Reply:
> @dduportal do you have something in place to receive them yourself or do you rely on the SECURITY tracker?

I confirm that we expect reporters to use the SECURITY tracker

Reply:
As pointed by @daniel-beck , the infra team need to fix the duplication and non-coherent informations then.